staging/src/k8s.io/client-go/examples/third-party-resources/main.go

```diff
@@ -52,24 +50,29 @@ func main() {
 	}
 
 	// initialize third party resource if it does not exist
-	tpr, err := clientset.ExtensionsV1beta1().ThirdPartyResources().Get("example.k8s.io", metav1.GetOptions{})
+	tpr, err := clientset.ExtensionsV1beta1().ThirdPartyResources().Get(ExampleResourceName, metav1.GetOptions{})
 	if err != nil {
 		if errors.IsNotFound(err) {
 			tpr := &v1beta1.ThirdPartyResource{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: "example.k8s.io",
+					Name: ExampleResourceName,
 				},
 				Versions: []v1beta1.APIVersion{
-					{Name: "v1"},
+					{Name: ExampleResourceVersion},
 				},
-				Description: "An Example ThirdPartyResource",
+				Description: ExampleResourceDescription,
 			}
 
 			result, err := clientset.ExtensionsV1beta1().ThirdPartyResources().Create(tpr)
 			if err != nil {
 				panic(err)
 			}
 			fmt.Printf("CREATED: %#v\nFROM: %#v\n", result, tpr)
+
+			// See the issue https://github.com/kubernetes/features/issues/95
+			// ("Make new TPRs available immediately after the create request succeeds")
+			fmt.Print("Sleeping to make sure the TPR is processed and available")
```

Comment:
How about upgrading the comment, if not the printed message, to point to the existing issue (https://github.com/kubernetes/features/issues/95) that notes the need for a delay (among many other things, sigh)?